api/src/lib/github/kubernetes.go

```diff
@@ -0,0 +1,40 @@
+package github
+
+import (
+	"regexp"
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+var portRegex = regexp.MustCompile("servicePort: ([0-9]{4})")
+
+func FetchK8Port(repository RepositoryResponse) ([] string)  {
+	files, err := FetchFolder(repository.Url, "kube/core")
+	if err != nil{
+		return []string{}
+	}
+
+	ports := make([]string,0)
```

Comment:
In the case of `ports`, you actually know what you think will be the size of this slice prior to creating it (size of `files`, so to avoid unnecessary resizes upon insertions you can pass in the capacity as a third argument (see https://tour.golang.org/moretypes/13 for example)